test/tests/dampers/max_increment/tests

```diff
@@ -5,4 +5,11 @@
     expect_out = '9\s*Nonlinear'
     absent_out = '13\s*Nonlinear'
   [../]
+  [./testverifydamping_percentage]
+    type = 'RunApp'
+    input = 'max_increment_damper_test.i'
```

Comment:
These two tests are likely to run at the same time, which means that if they are outputting a file, it'll likely get corrupted. While this isn't strictly a problem since you aren't reading the output files, we've run into enough problems with doing this that we try to avoid it. I'd suggest either turning off output in that file (either by editing the file or supplying command lines overrides on one of them to turn it off. The other option is to serialize these two tests by putting in a prereq.

Reply:
Ah, good to know. It's easy enough to just remove the entire outputs block.

Reply:
@permcody I removed the output block completely so they avoid corruption. That should resolve this yes?

Reply:
Yep! That takes care of it.